test/test-calculatorCoordinateOfTopLeft .js

```diff
@@ -0,0 +1,101 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris')
+describe("Coordinates next to each other and arrays is many elements, matches coordinate of one of the elements", function(){
+    it("Arrays is many elements", function(){
+        arrayOfCoordinates = tetris.getCoordinates(`
+            -----
+            --XX-
+            --XX-
+            -----
+            -----
+        `)
+        let topLeftCorner = calculatorCoordinateOfTopLeft(arrayOfCoordinates)
+        expectedResult = tetris.getCoordinates(`
+            -----
+            --X--
+            -----
+            -----
+            -----
+        `)
+    })
+})
+
+describe("Coordinates spaced apart, is 3 elements, does not match coordinate of any of the elements", function(){
```

Comment:
Why do you have one "it" block per "describe" block?
Why all "it" blocks are named "Arrays is many elements"?

Reply:
Please leave one describe and rename it blocks

Reply:
Sorry,  I don t understand you, what do you mean "Where are assert statements?"?

Reply:
Can I merge?

Reply:
I checked everything, I have done everything

Reply:
Done
"These are good.
Please add test where top left does not match any of the coordinates, like z-tetromino
Edit: i mean s-tetromino"

Reply:
Done 
"Where are assert statements?"

Reply:
1. All of these are incorrect
Add a test for square O-tetromino first - you should have no problems locating top left corner
DONE

2. why this coordinate?
DONE

3. you have single element, why the test sais otherwise?
DONE

4. why this coordinate?
    What tetromino are you talking about? arrayOfCoordinates is one coordinate
DONE	

5. These are good.
   Please add test where top left does not match any of the coordinates, like z-tetromino
   Edit: i mean s-
DONE

6. First element is matching left corner, please change
DONE
7. Why do you have spaces at the end of filename? Please remove
DONE
8. Why do you have one "it" block per "describe" block?
    Why all "it" blocks are named "Arrays is many elements"?
DONE

9. Please leave one describe and rename it blocks
DONE
10. Where are assert statements?
DONE
11. Please fix all comments
Maybe done

Reply:
"Why do you have spaces at the end of filename? Please remove"
I understand it and did this.
I forgot push this, sorry...
Can I merge?

Reply:
In the last
"In which commit did you do it?"